stable/mission-control/README.md

```diff
@@ -243,6 +243,23 @@ This can be done with the following parameters
 ...
 ```
 
+#### Elasticsearch with custom tls-certificates
+
+By default the internal elasticsearch uses the bundled tls-certificates for configuring searchguard. For production deployments it is recommended to use you own certificates.
+Custom certificates can be added by using kubernetes secret. The secret should be created outside of this chart and provided using the tag `.Values.elasticsearch.certificatesSecretName`. Please refer the example below.
```

Comment:
Provide an example how to create the necessary secret similar to the command below. Not all users understand how to make k8s secret the proper way.